cli/command/load_balancer_test.go

```diff
@@ -0,0 +1,40 @@
+package command
+
+import (
+	"testing"
+
+	"github.com/quintilesims/layer0/common/models"
+	"github.com/urfave/cli"
+)
+
+func TestCreateLoadBalancer(t *testing.T) {
+	base, ctrl := newTestCommand(t)
+	defer ctrl.Finish()
+
+	loadBalancerCommand := NewLoadBalancerCommand(base.Command())
+	c := &cli.Context{}
```

Comment:
use the `getCLIContext` helper function (in `test_helpers.go`) to generate your context. It takes positional args and flags to generate the context. For example:
```
// Args is an alias for []string 
args := Args{"env", "name"}

// Flags is an alias for map[string]interface{}
flags := Flags{
    "public": true,
    "port": []string{"80:80/tcp", "443:443/https"},
    "healthcheck-target": "tcp:80", 
    ...
}

c := getCLIContext(t, args, flags)
command := NewLoadBalancerCommand(base.Command())
if err := command.Create(c); err != nil {
    t.Fatal(err)
}
```